example/lib/pages/marker_rotate.dart

```diff
@@ -0,0 +1,161 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_map/flutter_map.dart';
+import 'package:latlong2/latlong.dart';
+
+import '../widgets/drawer.dart';
+
+class MarkerRotatePage extends StatefulWidget {
+  static const String route = '/marker_rotate';
+  @override
+  MarkerRotatePageState createState() {
+    return MarkerRotatePageState();
+  }
+}
+
+class MarkerRotatePageState extends State<MarkerRotatePage> {
+  bool rotateMarkerLondon;
+  bool rotateMarkerDublin;
+  bool rotateMarkerParis;
+  bool rotateMarkerLayerOptions = false;
```

Comment:
Markers are rotating when `rotateMarkerLayerOptions==false` 
![device-2021-04-07-100001](https://user-images.githubusercontent.com/866528/113831893-60c80600-9788-11eb-976a-5bea005235bb.png)

Reply:
What you see is correct. 
Markers are not rotating **relative** to the map if rotateMarkerLayerOptions is set to false.

Reply:
Because this is the default behaviour right now, I didn't changed it.
It would be a breaking change otherwise.